python/lsst/pipe/tasks/processBrightStars.py

```diff
@@ -211,34 +204,130 @@ class ProcessBrightStarsTask(PipelineTask):
+        extract stamps around each, then preprocess them.
+
+        Bright star preprocessing steps are: shifting, warping and potentially
+        rotating them to the same pixel grid; computing their annular flux,
+        and; normalizing them.
+
+        Parameters
+        ----------
+        inputExposure : `~lsst.afw.image.ExposureF`
+            The image from which bright star stamps should be extracted.
+        refObjLoader : `~lsst.meas.algorithms.ReferenceObjectLoader`, optional
+            Loader to find objects within a reference catalog.
+        dataId : `dict` or `~lsst.daf.butler.DataCoordinate`
+            The dataId of the exposure (including detector) that bright stars
+            should be extracted from.
+        skyCorr : `~lsst.afw.math.backgroundList.BackgroundList`, optional
+            Full focal plane sky correction obtained by `SkyCorrectionTask`.
+
+        Returns
+        -------
+        brightStarResults : `~lsst.pipe.base.Struct`
+            Results as a struct with attributes:
+
+            ``brightStarStamps``
+                (`~lsst.meas.algorithms.brightStarStamps.BrightStarStamps`)
+        """
+        self.config = cast(ProcessBrightStarsConfig, self.config)
+
+        if self.config.doApplySkyCorr:
+            self.log.info("Applying sky correction to exposure %s (exposure modified in-place).", dataId)
+            self.applySkyCorr(inputExposure, skyCorr)
+
+        self.log.info("Extracting bright stars from exposure %s", dataId)
+        # Extract stamps around bright stars.
+        extractedStamps = self.extractStamps(inputExposure, refObjLoader=refObjLoader)
+        if not extractedStamps.starStamps:
+            self.log.info("No suitable bright star found.")
+            return None
+        # Warp (and shift, and potentially rotate) them.
+        self.log.info(
+            "Applying warp and/or shift to %i star stamps from exposure %s.",
+            len(extractedStamps.starStamps),
+            dataId,
+        )
+        warpOutputs = self.warpStamps(extractedStamps.starStamps, extractedStamps.pixCenters)
+        warpedStars = warpOutputs.warpedStars
+        xy0s = warpOutputs.xy0s
+        brightStarList = [
+            BrightStarStamp(
+                stamp_im=warp,
+                archive_element=transform,
+                position=xy0s[j],
+                gaiaGMag=extractedStamps.gMags[j],
+                gaiaId=extractedStamps.gaiaIds[j],
+                minValidAnnulusFraction=self.config.minValidAnnulusFraction,
+            )
+            for j, (warp, transform) in enumerate(zip(warpedStars, warpOutputs.warpTransforms))
+        ]
+        # Compute annularFlux and normalize
+        self.log.info(
+            "Computing annular flux and normalizing %i bright stars from exposure %s.",
+            len(warpedStars),
+            dataId,
+        )
+        # annularFlux statistic set-up, excluding mask planes
+        statsControl = StatisticsControl()
```

Comment:
Making this change here and elsewhere results in lines that are too long, requiring it to be spaced across multiple lines. In some instances, the line length doesn't change, but I personally find it less readable than having individual configs set on their own line. After chatting with Amir about it, we've kept the older format for now. If you feel strongly, please do let us know.

Reply:
I'm confused. I thought it would just be:
```
statsControl = StatisticsControl(
    numsigmaClip=self.config.numSigmaClip,
    numIter=self.config.numIter,
)
```

Reply:
Changed usage of `StatisticsControl` here and elsewhere to the format you suggest.